static/app/components/breadcrumbs.tsx

```diff
@@ -102,8 +103,19 @@ const BreadcrumbList = styled('div')`
   padding: ${space(1)} 0;
 `;
 
-const BreadcrumbLink = styled(({preserveGlobalSelection, ...props}) =>
-  preserveGlobalSelection ? <GlobalSelectionLink {...props} /> : <Link {...props} />
+type BreadcrumbListProps = {
```

Comment:
should we name this type "BreadcrumbLinkProps"?

Reply:
yeah oops